pkg/asset/imagebased/configimage/configiso.go

```diff
@@ -0,0 +1,134 @@
+package configimage
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
```

Comment:
This package is deprecated, you'd be nice not to use it in new code.

Reply:
Thanks for catching this, I have updated this :pray: